lib/dont_use_recipient.rb

```diff
@@ -0,0 +1,62 @@
+require "httparty"
```

Comment:
Looks like you got started on this but didn't get finished.  I suggest removing it from git.